### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by hairy-tortoise** [DanTup/github-codespaces-flutter-web#3]
When I start a codespace, launch the app and go to the exposed port I get the Flutter loading, but it never finishes. I don't get any errors in the console.

**Comment by DanTup** [comment]
Yeah, this doesn't seem to work for me anymore. When I click the Dart Debug Extension button I start getting this notification appearing:

![Screenshot 2024-04-18 104908](https://github.com/DanTup/github-codespaces-flutter-web/assets/1078012/5a91d093-7c31-4187-923a-d0a39240650e)

And it just keeps appearing every few seconds, even after I've closed the app window.

I have all ports set to public:

![Screenshot 2024-04-18 104914](https://github.com/DanTup/github-codespaces-flutter-web/assets/1078012/cd9de1c6-dc75-4a46-b3f6-2df6d1811918)

@elliette do you know what this error is? It's not very clear from the notification I see, and I can't find any more details in any of the consoles for the app or by trying to inspect the extension (there used to be a way to view the "background page" I think, but now I only see a service worker link, and the console there is clear).

(this isn't a priority, but it'd be good to understand what's broken.. it's very possible it's something on codespaces, but the error isn't helpful)

**Comment by elliette** [comment]
Ah yes, this was added to fix a potential security vulnerability in the extension (details here: https://github.com/dart-lang/webdev/issues/2287)

**Comment by DanTup** [comment]
@elliette oh, I see - so do all domains need whitelisting now? Is it possible for the user to add their own (or could we prompt to ask them, rather than just blocking)? (again, not a priority - but would be a shame to lose the ability to use this on services like Codespaces).

**Comment by DanTup** [comment]
There's an open issue at https://github.com/dart-lang/webdev/issues/2375 tracking support for custom domains that aren't localhost.
